### ISSUE TITLE
Loss Functions: rotation matrix instead of axis-angle

### ISSUE THREAD

**Issue by ChenyuGao** [mohomran/neural_body_fitting#10]
in your paper Sec. 3.3.,  "we observed better performance by imposing the loss on the rotation matrix representation of θ rather than on its ‘native’ axis angle encoding as defined in SMPL".
And in the paper "Unite the People", it also said "It turned out to be critical to use full rotation matrices as regression targets: the axis-angle representation has **discontinuities**, adding noise to the loss function."
Do you know how to understand the discontinuities? On the contrary I think the axis-angle has continuities, after all, the rotation matrix is calculated by the axis-angle's trigonometric function.

**Comment by tszhang97** [comment]
same question. what's the difference betweem  axis-angle and  rotation matrix

**Comment by ghost** [comment]
I have same problem with this! I still wonder why the network was predict 226 SMPL parameters (Sec. 4.2, Architecture:) instead of 82. I think it use rotation in every pose parameter because 226 = 72 (pose including a rotation) * 3 + 10 (shape).

please explain it? @mohomran 

or you guys have any ideas about it? @willie1997 @GaoChenyuchn

**Comment by ghost** [comment]
Maybe there is a possible explanation. 
"Alternatively, we can transform the θ parameters from axis-angle representation to rotation matrix using the Rodrigues’ rotation formula, and apply an L2 loss on this representation instead (line 2). This leads to more stable training and better performance, as has also been observed by Lassner et al. However,"
<Learning to Estimate 3D Human Pose and Shape from a Single Color Image>, Sec. 5.3.
